Remarkable-Donkeys/bangazonWebApp
# Jf category seed

**The Name of this branch is jf-category-seed**


* **Please check if the PR fulfills these requirements**


* **What kind of change does this PR introduce?** (Bug Fix, Feature, Document Update...)
- properly seeds category table if the category table has 0 rows



* **What Ticket is associated with this PR**
None


* **What is the new behavior?**  (if this is a feature change)
- seeds category table

* **List all core functionality**


* **Other information / Steps to Test**:
1. run the following in the terminal
```
git fetch --all
git checkout jf-category-seed
```

2. run the application

3. close the application

4. check the category table in the sql server explorer and the table should be populated  

***At least one person needs to approve pull request before merged to master. Indicate if you think more than one person should test the feature***
